pom.xml

```diff
@@ -76,5 +76,17 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>cloud.orbit</groupId>
+            <artifactId>orbit-spring</artifactId>
+            <version>1.0.2</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>cloud.orbit</groupId>
+            <artifactId>orbit-hk2</artifactId>
```

Comment:
Orbit HK2 is deprecated. I would prefer not including it as a dependency going forward.